repooler.py

```diff
@@ -297,15 +297,18 @@ def integrate_conc_diff(lane_maps, desired_ratios):
         for index in xrange(1, len(lane_maps[key])):
             expect_output.append(1/float(len(lane_maps[key]) -1))#-1 Removes undetermined
         #TODO: One could include undetermined here
-        conc_factor[key] = actual_output/expect_output
+        #Overriding errstate since expected is 0 for undetermined
+        with numpy.errstate(divide='ignore'):
```

Comment:
So, if I understand this correctly. Here you want to avoid raising `ZeroDivisionError` and prefer `Inf` (or possibly `NaN`)?